Gruvo/Gruvo/ClientApp/src/app/signup-form/signup-form.component.html

```diff
@@ -0,0 +1,24 @@
+<div class="signup">
+  <h1>Sign Up</h1>
+  <div class="form-group">
+    <label for="Login">Login:</label>
+    <input id="Login" [(ngModel)]="login" type="text" class="form-control" placeholder="Login" maxlength="50">
```

Comment:
is this validation of max length corresponds with validation on server/db or other reasons?